port/freebsdamd64/omrsignal_context.c

```diff
@@ -0,0 +1,323 @@
+/*******************************************************************************
+ * Copyright (c) 1991, 2015 IBM Corp. and others
```

Comment:
The copyright header should have the current year: `Copyright (c) 1991, 2018 IBM Corp. and others`.